src/ChirpInfrastructure/CheepRepository.cs

```diff
@@ -24,18 +24,22 @@ public class CheepRepository(ChirpDBContext context) : ICheepRepository
 
   }
   Above will be relevant later*/
-  public List<CheepDTO> ReadCheeps()
+  public List<CheepDTO> ReadCheeps(int pageNumber)
   {
-	var cheepList = new List<CheepDTO>();
-
+	//var cheepList = new List<CheepDTO>();
+	int pageSize = 32;
 	//query for getting every cheep
 	var query = _context.Cheeps.Select(message => new CheepDTO() // message = domain cheep. result = cheepDTO
 	{
 	  Text = message.Text,
 	  AuthorID = message.AuthorId,
 	  AuthorName = message.Author.Name,
 	  TimeStamp = message.TimeStamp.ToString("MM/dd/yy H:mm:ss") 
-	}).OrderBy(message => message.AuthorID); // As long as TimeStamp.ToString("") is present then we can't sort cheeps by time.
+	})
```

Comment:
Is there a particular reason for this line break?

Reply:
Not particularly, just for a better code structure. 
Just moved .OrderBy down to be above to .Skip and .Take